queue/queue.py

```diff
@@ -3,13 +3,28 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
+  #enqueue should add an item to the back of the queue.
   def enqueue(self, item):
-    pass
+    self.storage.append(item)
   
+  #should remove and return an item from the front of the queue
   def dequeue(self):
-    pass
+    #check if there are items
+    if not self.storage:
+      return None
+    #remove item at first position
+    remove_item = self.storage[0]
 
+    self.storage.remove(self.storage[0])
+    return remove_item
+
```

Comment:
Oh, very good here, especially if this is an array implementation of a queue. I think, though, if you do it this way, it's still constant time, so that's a plus.

However, if the objective is to use a Linked List, you should check out the `remove_head()` method!